plaidml2/core/ffi.cc

```diff
@@ -62,6 +85,8 @@ void plaidml_shutdown(plaidml_error* err) {
   ffi_wrap_void(err, [&] {
     IVLOG(1, "plaidml_shutdown");
     GetPlatform().platform.reset(nullptr);
+    auto ctx = GlobalContext::getContext();
+    ctx->eventlog()->FlushAndClose();
```

Comment:
Let's reset the eventlog to `nullptr` here so that internally it gets deleted (I think this is causing CI tests to crash)

Reply:
Also thinking that we should explicitly use `atexit` in the C++ code so that C++ users will also benefit from the eventlog being flushed.